oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDefinition.java

```diff
@@ -79,17 +79,14 @@
     public final long bulkFlushIntervalMs;
     public final int bulkRetries;
     public final long bulkRetriesBackoff;
-    private final String indexPrefix;
     private final String remoteAlias;
 
     private final Map<String, List<PropertyDefinition>> propertiesByName;
 
     public ElasticIndexDefinition(NodeState root, NodeState defn, String indexPath, String indexPrefix) {
-        super(root, getIndexDefinitionState(defn), determineIndexFormatVersion(defn), determineUniqueId(defn), indexPath);
-        boolean isReindex = defn.getBoolean(IndexConstants.REINDEX_PROPERTY_NAME);
-        String indexSuffix = "-" + (getReindexCount() + (isReindex ? 1 : 0));
-        this.indexPrefix = indexPrefix != null ? indexPrefix : "";
-        this.remoteAlias = ElasticIndexNameHelper.getIndexAlias(indexPrefix, getIndexPath());
+        super(root, defn, determineIndexFormatVersion(defn), determineUniqueId(defn), indexPath);
+        String indexSuffix = "-" + getReindexCount();
```

Comment:
I'm not quite sure if the getReindexCount() now makes sense here... we will see! If tests start failing, then maybe it doesn't work any longer.

Reply:
We still need some suffix to handle re-indexing transparently. As discussed we need to replace `reindexCount` with `seed` here.